### PR TITLE
chore: optimize the format of the generated document

### PR DESCRIPTION
**Pull request by nic-chen**
optimize the format of the generated document for a better display

### CODE REVIEW

**Review by kezhenxu94 — APPROVED** [review]
LGTM

**Review by wu-sheng — APPROVED** [review]

File: docs/plugin-description.md [hunk 1]
```diff
@@ -55,27 +55,27 @@ insecure_skip_verify: true
 # api.Fallbacker
 ## timer-fallbacker
 ### description
-```this is a timer fallback trigger when forward fails.```
+this is a timer fallback trigger when forward fails.
 ### defaultConfig
-```
+```yaml
```
Comment (membphis) [inline docs/plugin-description.md:60]: we'd better add a blank line between title and code block

**Review by gxthrj — APPROVED** [review]
LGTM

File: docs/plugin-description.md (hunk 1 above)
Comment (nic-chen) [reply docs/plugin-description.md:60]: @membphis  
not sure about that , but I found it doesn't have blank line here:
https://www.markdownguide.org/extended-syntax/#syntax-highlighting
please have a look. thanks.

Comment (kezhenxu94) [reply docs/plugin-description.md:60]: It's not mandatory in terms of syntax, just personal preference IMO

**Comment by EvanLjp** [comment]
LGTM

**Review by EvanLjp — APPROVED** [review]

**Review by moonming — APPROVED** [review]

**Pull request merged**